### ISSUE TITLE
Progress Bar Hangs at 99%

### ISSUE THREAD

**Issue by Scott1941** [DYefremov/FFmpegGTK#2]
I really like your application and it works perfectly on MX Linux. In fact, I personally think it's better than WinFF.

I realise that it was developed for Linux but I would really like to use it on my OpenBSD machine (WinFF and QWinFF are not in the OpenBSD repo).

It actually works quite well on OpenBSD and has successfully transcoded all the media I have tried so far. There is just one small issue on OpenBSD. The progress bar hangs on 99%, even though the conversion job has actually completed. Is there some simple edit to the python script, or anything else, that might fix this issue? Thanks for any help or ideas.

**Comment by DYefremov** [comment]
Hi,
Is the ''Done!" message displayed after the conversion is completed?
![done](https://github.com/DYefremov/FFmpegGTK/assets/7511379/661da393-e579-49d8-b28f-50aab1a0838b)
If not, then you should pay attention to [this code section](https://github.com/DYefremov/FFmpegGTK/blob/0f0e6cb307d534df3899db26f46218233a6f1d1d/app/converter.py#L441). Perhaps the _subprocess_ module works a little differently in OpenBSD. If possible, I will try to experiment with OpenBSD in a virtual machine.

**Comment by Scott1941** [comment]
Hi

Thank you for replying so quickly.

No, on OpenBSD the "done" message is not displayed. It only works for
Linux.

However, the application seems to convert media correctly.

Regards
Scott

**Comment by DYefremov** [comment]
I experimented a little on a virtual machine. I have added a small fix (available in the [development](https://github.com/DYefremov/FFmpegGTK/tree/development) branch). Unfortunately this is more a little hack. It may need to be improved. Please check both systems (Linux and BSD).

**Comment by Scott1941** [comment]
Thanks Dmitriy, I will test it out and get back to you ASAP with my
findings.

**Comment by Scott1941** [comment]
Hi Dmitriy,

I have tried the development branch on my MX Linux desktop and my 2 OpenBSD
laptops. I have converted individual mp4 and webm files to avi and mp3.
I have also done multiple batch conversions of mp4 and webm files to avi
and mp3. Everything so far has worked perfectly. Thank you. The progress
bar goes to 100% and the "Done!" message appears.

Will the changes be merged into the main branch, or will I just run the
development branch on OpenBSD? Thanks again for your help.

Kind Regards
Scott
